modules/nf-core/ilastik/multicut/main.nf

```diff
@@ -0,0 +1,38 @@
+process ILASTIK_MULTICUT {
+    tag "$meta.id"
+    label 'process_low'
+
+    container "labsyspharm/mcmicro-ilastik:1.6.0"
+
+    input:
+    tuple val(meta), path(h5)
+    path ilp
+    path probs
+
+    output:
+    tuple val(meta), path("*.tiff") , emit: out_tiff
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    /ilastik-release/run_ilastik.sh \\
+        --headless \\
+        --readonly 1 \\
+        --project=$ilp \\
+        --raw_data=$h5 \\
+        --probabilities=$probs \\
+        --export_source="Multicut Segmentation" \\
```

Comment:
Same here with `--export_source`

Reply:
So for Multicut, this is the only `--export_source` definition allowed and thus is the mandatory default argument, which still has to be provided as far as I understand the documentation:
https://www.ilastik.org/documentation/basics/headless

![image](https://user-images.githubusercontent.com/11639154/220344831-b117b0c7-d669-4887-8373-dff1991cce4d.png)

Reply:
That's strange they implemented it like this... But yeah it's fine if the arg is mandatory